.gitignore

```diff
@@ -5,3 +5,4 @@ composer.lock
 phpunit.phar
 phpunit.xml
 modd.conf
+.idea
```

Comment:
Can you please remove this rule? Any non-related file or folder such as the ones generated by your IDE must be placed always in your global Git Ignore file instead of adding them on a project basis. If not, all projects could end adding many folders to exclude the IDE settings such as `.idea` for PHP Storm or `nbproject` for Netbeans just to add 2 examples.

Reply:
roger that. Thanks 😄